### PR TITLE
reorganized files by moving to a new directory

### PR DESCRIPTION
**Pull request by Vandana-Rajamani**
## oracle/nosql-examples

Consolidated all sample schemas(BaggageInfo,AcctStreamSchema,GeoJSONSchema) and put them in a new directory SampleSchemas

### CODE REVIEW

**Comment by Vandana-Rajamani** [comment]
Added new directories for sqlexamples

File: examples-nosql-dotnet-sdk/sqlexamples/AddData.cs [hunk 1]
```diff
@@ -0,0 +1,321 @@
+/*-
+ * Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.
```
Comment (mikebrey) [inline examples-nosql-dotnet-sdk/sqlexamples/AddData.cs:2]: correct copyright years

File: examples-nosql-dotnet-sdk/sqlexamples/CreateTable.cs [hunk 2]
```diff
@@ -0,0 +1,82 @@
+/*-
+ * Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.
```
Comment (mikebrey) [inline examples-nosql-dotnet-sdk/sqlexamples/CreateTable.cs:2]: correct copyright year

File: examples-nosql-dotnet-sdk/sqlexamples/QueryData.cs [hunk 3]
```diff
@@ -0,0 +1,340 @@
+/*-
+ * Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.
```
Comment (mikebrey) [inline examples-nosql-dotnet-sdk/sqlexamples/QueryData.cs:2]: correct copyright year

File: examples-nosql-go-sdk/sqlexamples/AddData.go [hunk 4]
```diff
@@ -0,0 +1,320 @@
+package main
```
Comment (mikebrey) [inline examples-nosql-go-sdk/sqlexamples/AddData.go:1]: needs copyright and licensing

File: examples-nosql-go-sdk/sqlexamples/CreateTable.go [hunk 5]
```diff
@@ -0,0 +1,72 @@
+package main
```
Comment (mikebrey) [inline examples-nosql-go-sdk/sqlexamples/CreateTable.go:1]: needs copyright and licensing

File: examples-nosql-go-sdk/sqlexamples/QueryData.go [hunk 6]
```diff
@@ -0,0 +1,360 @@
+package main
```
Comment (mikebrey) [inline examples-nosql-go-sdk/sqlexamples/QueryData.go:1]: needs copyright and licensing

File: examples-nosql-java-sdk/sqlexamples/AddData.java [hunk 7]
```diff
@@ -0,0 +1,332 @@
+import java.io.File;
```
Comment (mikebrey) [inline examples-nosql-java-sdk/sqlexamples/AddData.java:1]: needs copyright and licensing

File: examples-nosql-java-sdk/sqlexamples/CreateTable.java [hunk 8]
```diff
@@ -0,0 +1,75 @@
+import java.io.File;
```
Comment (mikebrey) [inline examples-nosql-java-sdk/sqlexamples/CreateTable.java:1]: needs copyright and licensing

File: examples-nosql-java-sdk/sqlexamples/QueryData.java [hunk 9]
```diff
@@ -0,0 +1,346 @@
+import java.io.File;
```
Comment (mikebrey) [inline examples-nosql-java-sdk/sqlexamples/QueryData.java:1]: needs copyright and licensing

File: examples-nosql-node-sdk/sqlexamples/AddData.js [hunk 10]
```diff
@@ -0,0 +1,304 @@
+'use strict';
```
Comment (mikebrey) [inline examples-nosql-node-sdk/sqlexamples/AddData.js:1]: needs copyright and licensing

File: examples-nosql-node-sdk/sqlexamples/CreateTable.js [hunk 11]
```diff
@@ -0,0 +1,64 @@
+'use strict';
```
Comment (mikebrey) [inline examples-nosql-node-sdk/sqlexamples/CreateTable.js:1]: needs copyright and licensing

File: examples-nosql-node-sdk/sqlexamples/QueryData.js [hunk 12]
```diff
@@ -0,0 +1,322 @@
+'use strict';
```
Comment (mikebrey) [inline examples-nosql-node-sdk/sqlexamples/QueryData.js:1]: needs copyright and licensing

File: examples-nosql-python-sdk/sqlexamples/AddData.py [hunk 13]
```diff
@@ -0,0 +1,281 @@
+import os
```
Comment (mikebrey) [inline examples-nosql-python-sdk/sqlexamples/AddData.py:1]: needs copyright and licensing

File: examples-nosql-python-sdk/sqlexamples/CreateTable.py [hunk 14]
```diff
@@ -0,0 +1,41 @@
+import os
```
Comment (mikebrey) [inline examples-nosql-python-sdk/sqlexamples/CreateTable.py:1]: needs copyright and licensing

File: examples-nosql-python-sdk/sqlexamples/QueryData.py [hunk 15]
```diff
@@ -0,0 +1,293 @@
+import os
```
Comment (mikebrey) [inline examples-nosql-python-sdk/sqlexamples/QueryData.py:1]: needs copyright and licensing

**Review by mikebrey — CHANGES_REQUESTED** [review]
see comments for changes

**Comment by Vandana-Rajamani** [comment]
added/corrected copyright and license info in all files

File: examples-nosql-dotnet-sdk/sqlexamples/AddData.cs [hunk 16]
```diff
@@ -1,6 +1,5 @@
-/*-
- * Copyright (c) 2018, 2021 Oracle and/or its affiliates. All rights reserved.
- *
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates. All rights reserved.
```
Comment (mikebrey) [inline examples-nosql-dotnet-sdk/sqlexamples/AddData.cs:2]: change copyright to (c) 2022, 2023 
Change on all .NET examples

File: examples-nosql-java-sdk/sqlexamples/AddData.java [hunk 17]
```diff
@@ -1,3 +1,6 @@
+/*Copyright (c) 2023 Oracle and/or its affiliates.
```
Comment (mikebrey) [inline examples-nosql-java-sdk/sqlexamples/AddData.java:1]: change JAva SDK examples to (c) 2020, 2023

File: examples-nosql-python-sdk/sqlexamples/AddData.py [hunk 18]
```diff
@@ -1,3 +1,5 @@
+# Copyright (c) 2023 Oracle and/or its affiliates.
```
Comment (mikebrey) [inline examples-nosql-python-sdk/sqlexamples/AddData.py:1]: change python examples to (c) 2019, 2023

**Review by mikebrey — CHANGES_REQUESTED** [review]
additional modifications to the copyrights.

**Comment by Vandana-Rajamani** [comment]
updated copyrights info

**Review by mikebrey — APPROVED** [review]

**Pull request merged**